static/js/components/request.vue

```diff
@@ -136,6 +136,21 @@ export default {
         return 'IMAGE';
       }
     },
+    slitWidthToExposureTime: function(slitWidth){
```

Comment:
https://www.w3schools.com/js/js_switch.asp

Reply:
thanks for the code shaming, I'll change it

Reply:
to use a switch statement, I would have to hardcode in the exact values for the slits to compare against (can't use .include()). Do you think that is preferable here?

Reply:
It was just a suggestion